mypy.ini

```diff
@@ -0,0 +1,10 @@
+[mypy]
+
+[mypy-boto.*]
```

Comment:
These config sections tell Mypy to ignore third-party packages that are missing type hints.

Reply:
I added a comment to the file for future reference.